Lib/gflanguages/data/languages/de_Latn.textproto

```diff
@@ -33,11 +33,11 @@ region: "SK"
 region: "US"
 exemplar_chars {
   base: "a ä b c d e f g h i j k l m n o ö p q r s ß t u ü v w x y z"
-  auxiliary: "á à ă â å ã ā æ ç é è ĕ ê ë ē ğ í ì ĭ î ï İ ī ı ñ ó ò ŏ ô ø ō œ ş ú ù ŭ û ū ÿ"
+  auxiliary: "á à ă â å ã ā æ ç é è ĕ ê ë ē ğ í ì ĭ î ï İ ī ı ñ ó ò ŏ ô ø ō œ ş ſ ú ù ŭ û ū ÿ"
   marks: "◌̈ ◌̀ ◌́"
-  numerals: "- ‑ , . % ‰ + 0 1 2 3 4 5 6 7 8 9"
-  punctuation: "- ‐ ‑ – — , ; : ! ? . … \' ‘ ‚ \" “ „ « » ( ) [ ] { } § @ * / & #"
-  index: "A B C D E F G H I J K L M N O P Q R S T U V W X Y Z"
+  numerals: "- , . % + 0 1 2 3 4 5 6 7 8 9"
+  punctuation: "- – — , ; : ! ? . … \' ‘ ‚ \" “ „ « » ( ) [ ] { } @ * / & #"
+  index: "A B C D E F G H I J K L M N O P Q R S ẞ T U V W X Y Z"
```

Comment:
ẞ shouldn’t be in the de_Latn index it isn’t sorted differently than SS.

Reply:
Can we solve it by adding all the capitals in the base glyphs including capital eszett?

Reply:
Yes, the capitals should be added to the base characters. The index is for indexing/sorting, unfortunately there’s no explanation on what it’s for here. I don’t think GF uses the index anywhere, it was imported from the CLDR which uses it for sorting. We need a different PR that adds the uppercase characters, it was done previously for some language data in #53 but not for all.